### PR TITLE
Added the v3 endpoint for newsleters_subscriptions

### PR DESCRIPTION
**Pull request by santostiago**
This is a very simple feature, as it's just copying it from V2.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -15,6 +15,7 @@
         resources :columns, only: [:index]
       end
       get '/get_all_nodes', to: 'nodes#get_all_nodes'
+      resources :newsletter_subscriptions, only: [:create]
```
Comment (agnessa) [inline config/routes.rb]: maybe don't remove it from v2 just yet, so that we don't break anything in the frontend?

Comment (santostiago) [reply config/routes.rb]: Yes, I know. I didn't remove it. The routes were duplicated, so I just removed one of them.
It still exists in line 24.

**Comment by tiagojsag** [comment]
no tests??????

**Comment by santostiago** [comment]
@tiagojsag I've done the tests, but I removed them as I found them pointless.
For the test to pass, there should be a valid mail chimp API key. And it would actually add the email provided to the mailing list (as this is the only thing this endpoint does). So I thought it would be better to remove the test.
@agnessa I've updated the swagger file.

**Pull request merged**